### ISSUE TITLE
gc_pthread_redirects.h not installed

### ISSUE THREAD

**Issue by jengelh** [ivmai/bdwgc#391]
gc_pthread_redirects.h does not get installed when SINGLE_GC_OBJ is in effect, because the pkginclude_ line was (IMO erroneously) placed within !SINGLE_GC_OBJ.

```
if SINGLE_GC_OBJ

libgc_la_SOURCES = extra/gc.c

if PTHREAD_START_STANDALONE
AM_CPPFLAGS += -DGC_PTHREAD_START_STANDALONE
libgc_la_SOURCES += pthread_start.c
endif

else

...

# C Library: Architecture Dependent
# ---------------------------------

if WIN32_THREADS
libgc_la_SOURCES += win32_threads.c
else
if PTHREADS
# Not Cygwin or MinGW.
libgc_la_SOURCES += pthread_start.c pthread_support.c
if DARWIN_THREADS
libgc_la_SOURCES += darwin_stop_world.c
else
libgc_la_SOURCES += pthread_stop_world.c
endif
endif
endif
```

**Comment by jengelh** [comment]
(#389)

**Comment by ivmai** [comment]
Yes, it duplicates #388  and is already solved by #389
